services/ui-src/src/components/export/ExportedSectionHeading.tsx

```diff
@@ -11,38 +11,48 @@ export const ExportedSectionHeading = ({
   reportType,
   verbiage,
 }: Props) => {
-  const sectionHeading = verbiage?.intro?.exportSectionHeader
-    ? verbiage?.intro?.exportSectionHeader
-    : verbiage?.intro?.subsection || heading;
+  const sectionSubHeader = verbiage?.intro?.subsection || heading;
   const sectionInfo = verbiage?.intro?.exportSectionHeader
     ? null
     : verbiage?.intro?.info;
   const sectionSpreadsheet = verbiage?.intro?.spreadsheet;
+  const sectionInfoHeader: any =
+    verbiage?.intro?.info && verbiage?.intro?.info[0];
+  const sectionIntroType = sectionInfoHeader && sectionInfoHeader.type;
+  const sectionIntroContent = sectionInfoHeader && sectionInfoHeader.content;
 
   return (
-    <Box data-testid="exportedSectionHeading" sx={sx.container}>
-      <Heading as="h2" sx={sx.heading}>
-        {sectionHeading}
-      </Heading>
-      {sectionInfo && (
-        <Box sx={sx.info}>
-          <Text>
-            {typeof sectionInfo === "string"
-              ? sectionInfo
-              : parseCustomHtml(sectionInfo)}
-          </Text>
-        </Box>
-      )}
-      {sectionSpreadsheet && (
-        <Box sx={sx.spreadsheet}>
-          <SpreadsheetWidget
-            description={sectionSpreadsheet}
-            isPdf={true}
-            reportType={reportType}
-          />
-        </Box>
-      )}
-    </Box>
+    <>
+      {sectionSubHeader ? (
+        <Heading as="h3" sx={sx.heading.h3}>
+          {sectionIntroType === "heading" &&
```

Comment:
This conditional is _just_ long enough that I might consider pulling it out into a function, especially for the hard-coded `sectionIntroContent` checks...It might be nice to name it so it's clear why we are excluding just those ones.